packages/@aws-cdk-testing/cli-integ/resources/cdk-apps/app/app.js

```diff
@@ -422,6 +422,29 @@ class LambdaStack extends cdk.Stack {
   }
 }
 
+class SessionTagsLambdaStack extends cdk.Stack {
+  constructor(parent, id, props) {
+    super(parent, id, {
+      ...props,
+      synthesizer: new DefaultStackSynthesizer({
+        deployRoleSessionTags: {
+          'Department' : 'Engineering',
+        },
+        fileAssetPublishingRoleSessionTags: {
```

Comment:
Add (and assert on) session tags for the docker image upload role and add a docker image asset to this stack.